packages/repository-tests/src/crud/relations/fixtures/models/product.model.ts

```diff
@@ -1,18 +1,23 @@
 // Copyright IBM Corp. 2019. All Rights Reserved.
-// Node module: @loopback/repository
+// Node module: @loopback/repository-tests
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {Entity, model, property} from '../../..';
+import {Entity, model, property} from '@loopback/repository';
 
-@model()
+@model({
+  settings: {
+    strictObjectIDCoercion: true,
+  },
+})
 export class Product extends Entity {
   @property({
-    type: 'number',
+    type: 'string',
```

Comment:
I am not very happy that we are using a string type that's later changed to a number. It makes it too easy to forget to update the type from a test that's using this fixture. I would really like to the fixtures structured in a way that allows us to leverage `features.idType`.